src/emailservice/email_server.py

```diff
@@ -122,7 +129,10 @@ def Check(self, request, context):
 
 def start(dummy_mode):
   server = grpc.server(futures.ThreadPoolExecutor(max_workers=10),
-                       interceptors=(tracer_interceptor,))
+                       interceptors=(oc_tracer_interceptor,))
+  # add OpenTelemetry interceptor
```

Comment:
The comment should not only about what is done but also why it's done.

Reply:
addressed